core/src/main/java/org/openstack4j/model/common/EventList.java

```diff
@@ -0,0 +1,38 @@
+package org.openstack4j.model.common;
+
+import org.openstack4j.common.Buildable;
+import org.openstack4j.model.ModelEntity;
+import org.openstack4j.model.common.builder.EventListBuilder;
+
+/**
+ * Model for the generic events
+ *
+ * @author sujit sah
+ *
+ */
+public interface EventList extends ModelEntity, Buildable<EventListBuilder> {
```

Comment:
The naming here is also confusing. See this signature `public List<? extends EventList> getEvents()`, it makes an impression of a list of lists, but it is not the case.

IIUC, those are event details. Does `ServerActionEvent` sound good to you as a class name?